### DISCUSSION TITLE
Whisper Auto Transcribe - An all-in-one solution for transcription

### DISCUSSION THREAD

**Discussion by tomchang25** [openai/whisper#1234] [Show and tell]
Hello everyone.
I have integrated Whisper into the Gradio framework and added a bunch of features.
Today, I have released the alpha version 3.0. 
Here are the new features in comparison to the original version:

### Unique feature:

- Includes a one-click installer
- Increased time precision from 1 to 0.01 seconds
- Supports Youtube integration
- Preview subtitles in video
- Provides support for Background Music Mute, works fine even during heavy metal live performances
- Supports long files, 3-hour files have been tested
- Resolves the issue of subtitle repetition

You can test it by running:
`git clone --branch v3-alpha https://github.com/tomchang25/whisper-auto-transcribe.git whisper-auto-transcribe-v3`

Also, don't forget to check out this project on [GitHub](https://github.com/tomchang25/whisper-auto-transcribe/tree/main)

**Comment by dgoryeo** [comment]
Hi @tomchang25 , are the checkboxes for VAD and Vocal_Extractor placeholder for future implementation or are they already functional opions? Thanks for the good initiaive!

**Reply by tomchang25** [reply]
Hello @dgoryeo, these two features are not support in v2.x. But it has been implemented in v3.0 now. You can either use

 `git clone --branch v3-alpha https://github.com/tomchang25/whisper-auto-transcribe.git whisper-auto-transcribe-v3`

 or check the v3.0-alpha branch on GitHub.
